### ISSUE TITLE
Add more images -- Enhance web pages

### ISSUE THREAD

**Issue by stumbo** [Interlisp/Interlisp.github.io#41]
Add more images to the existing documentation.  Show off the artifacts that have been generated. Make it easier for visitor to envision and respond to our mission.  Make the site more interesting.

**Comment by pamoroso** [comment]
I second this recommendation.

Here's a relevant anecdote about [STk](http://kaolin.unice.fr/STk), a Scheme implementation with Tk bindings. A while back I suggested the maintainer, Erick Gallesio, that he add some screenshots to the project site which had none. After adding [some screenshots](http://kaolin.unice.fr/STk/screenshots.html) he reported getting 6X more website visits.

**Comment by masinter** [comment]
just to enumerate the possibilities:
We could 
- make an email address: screenshots@interlisp.org -- a list or a person alias
- wire up the Hardcopy menu (@rmkaplan was talking about hardcopy to PDF via postscript)
- make a google group for sending screenshots medley-screenshots@googlegroups.com
- revive the Wiki for people to post things

The criteria for evaluation
- Ease of use
- Difficult to mususe
- admin review?

**Comment by masinter** [comment]
perhaps the simplest would be to make a GitHub Discussion. Discussions are like Issues but they aren't closed by commits etc. Earlier I took some Issues and made them Discussions but I wasn't clear on the distinction.

In the medley repo please.

**Comment by pamoroso** [comment]
Can I go ahead and create a pinned discussion in the `medley` repo along the lines discussed [here](https://github.com/Interlisp/Interlisp.github.io/issues/41#issuecomment-1456434712)?

**Comment by masinter** [comment]
Yes--LarryMasinter.netOn Mar 7, 2023, at 12:04 AM, Paolo Amoroso ***@***.***> wrote:﻿
Can I go ahead and create a pinned discussion in the medley repo along the lines discussed here?

**Comment by pamoroso** [comment]
I created the [pinned discussion](https://github.com/Interlisp/medley/discussions/1097) and [seeded it with a few screenshots](https://github.com/Interlisp/medley/discussions/1097#discussioncomment-5229693).

**Comment by pamoroso** [comment]
A couple of possible approaches to enhance the project site with more images are to add a few screenshots to some of the pages, and/or embed a photo gallery component to accommodate more (photo gallery add-ons for Hugo are available).

Also, the closest thing to the Smalltalk Zoo I can think of is a specialized version of Interlisp Online, with custom sysloads for showcasing specific features or greetfiles that set up the environment to script the demonstration of some tool or program.

Any approaches you prefer?

**Comment by nbriggs** [comment]
@pamoroso I don't think it needs to be as space-consuming as making new sysouts or greet files to demo things -- you can have a button to load a file into the normal system which sets up the environment as needed and scripts a demo.  You can drive the cursor programmatically and generate clicks and type-in -- I don't remember whether there are already aides for doing that, but Ron might know.  Moving the cursor with `\SETCURSORPOSITION` works.

**Comment by pamoroso** [comment]
Thanks, that's even better.

**Comment by masinter** [comment]
(i'm merging the remaining Interlisp.github.io issues into the medley issue list so we have one source... many of these issues might be resolved outside of the web site in any case.
